repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl

```diff
@@ -17,24 +18,16 @@
 <#-- Content Page Pixel -->
 <@contentPixel document.getCanonicalUUID() document.title></@contentPixel>
 
+<@headerBanner document />
```

Comment:
comparing this to `general-article.ftl` suggests that the banner image should be _inside_ the article, just above the grid-wrapper div

Reply:
You right, I was taking a look at the list-article which has it outside of the article.
I have pushed another version with it inside the article.

Reply:
Good spot, Harry.

Having said that: I know I kept mentioning `general-article.ftl` as the main reference, but having looked more closely into this with Andre, it seems that `nhsd-list-article.ftl` is probably a better basis for API catalogue page.

In `nhsd-list-article.ftl` the `<@headerBanner document />` is not wrapped in any container element whilst in `general-article.ftl`. Based on this, if the page looks okay in testing with the header wrapped in another container element I think it ultimately doesn't matter much and we can go ahead with either variant.